### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by vinc3nati**

### CODE REVIEW

File: badge.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+<body>
+    <!-- Avatar Badges -->
+    <div class="badge">
+        <img class="avatar" src="https://source.unsplash.com/3uuLWb6aQXc/" alt="user">
+        <div class="online"></div>
+    </div>
+    <div class="badge">
+        <img class="avatar" src="https://source.unsplash.com/3uuLWb6aQXc/" alt="user">
+        <div class="offline"></div>
```
Comment (Parul1999) [inline badge.html]: Suggestion - Use span here as well as used at other places in badge itself

File: badge.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <img class="avatar" src="https://source.unsplash.com/3uuLWb6aQXc/" alt="user">
+        <div class="dnd"></div>
+    </div>
+    <!-- Icon Badges -->
+    <div class="badge">
+        <ion-icon class="badge-icon" name="chatbubble"></ion-icon>
+        <span class="number">9</span>
+    </div>
+    <div class="badge">
+        <ion-icon class="badge-icon" name="bag"></ion-icon>
```
Comment (Parul1999) [inline badge.html]: Suggestion - use google fonts or if you are using any library do mention in your README

File: css/style.css [hunk 3]
```diff
@@ -95,3 +109,45 @@ body {
     outline: #7cc6fe solid 1px;
```
Comment (Parul1999) [inline css/style.css]: Don't hard-code colors . Use variables .

File: components/Alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+.alert.success {
+  background-color: var(--color-success);
+}
+.alert.info {
+  background-color: var(--color-info);
+}
+.alert.outline-primary {
+  outline: var(--color-primary-light) solid 1px;
+  color: var(--color-primary);
+  background-color: #f4faff;
```
Comment (Parul1999) [inline components/Alert/alert.css]: Use CSS variables , instead of hardcoding

**Review by Parul1999 — CHANGES_REQUESTED** [review]
Use CSS variables instead of hard-coding the codes.

File: badge.html (hunk 2 above)
Comment (vinc3nati) [reply badge.html]: I'm using google fonts , if you mean font awesome then it also requires cdn import.

**Review by anuj-kumary — CHANGES_REQUESTED** [review]
One suggestion your badge size is too small if your increase it's width and height it look nice

**Review by anuj-kumary — APPROVED** [review]
Looks Good !!

**Review by Shah-Arsalan — APPROVED** [review]

**Pull request merged**